### PR TITLE
Prepare for release v1.11.0-rc0

### PR DESCRIPTION
**Pull request by joestringer**
See the included `CHANGELOG.md` in the middle commit for a full list of changes.

### CODE REVIEW

**Comment by joestringer** [comment]
Smoke tests passed on v1.11.0-rc0 commit. Pushing revert commit.

**Review by christarazi — APPROVED** [review]

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -768,8 +768,8 @@ hubble:
       # -- Hubble-ui frontend image.
       image:
         repository: quay.io/cilium/hubble-ui
-        tag: latest
-        pullPolicy: Always
+        tag: v0.7.9@sha256:e0e461c680ccd083ac24fe4f9e19e675422485f04d8720635ec41f2ba9e5562c
```
Comment (aanm) [inline install/kubernetes/cilium/values.yaml]: We might want to use 0.8.0, waiting for @geakstr input.

**Review by aanm — APPROVED** [review]
LGTM overall. BTW shouldn't we keep the AUTHORS list updated and not revert that file?

**Comment by joestringer** [comment]
Latest version pushes the AUTHORS update into the .mailmap commit and also introduces a bump for the hubble ui/backend image tags to v0.8.2.

**Pull request merged**